test/kv_integration_test.go

```diff
@@ -149,6 +149,77 @@ func (kvSuite *KvTestSuite) TestAll() {
 
 }
 
+func (kvSuite *KvTestSuite) TestRangeScan() {
```

Comment:
* validate the schema was created successfully.
* more tests + negative
* negative tests that writes multiple frames to the same table